### PR TITLE
Paper

### PR DESCRIPTION
**Pull request by bobmyhill**
Opened for approval from all authors. 
Once everyone has approved, I'll submit to JOSS.

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+to estimate phase proportions is shown in \autoref{fig:mars_fit}.
+
+![Mineral phase proportions in the mantle of Mars, estimated from
+high pressure experimental data [@Bertka:1997]. Weighted residuals
+(misfits) are also shown, indicating that the majority of experimental
+run products are consistent with the reported starting composition.
+\label{fig:mars_fit}](figures/fit_mars_experiments.png)
+
+`BurnMan` does not attempt to replicate Gibbs minimization codes,
+of which there are many (PerpleX, HeFESTo, TheriakDomino, FactSAGE and MageMin).
```
Comment (gassmoeller) [inline paper/paper.md:150]: I think we need references for all codes we list by name. The point of JOSS is to improve the citation of codes and reviewers will likely be picky about this.

File: paper/paper.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+Instead, it provides two methods to deal with the problem of
+thermodynamic equilibrium: (1) reading in a P-T table of precalculated
```
Comment (gassmoeller) [inline paper/paper.md:152]:
```suggestion
thermodynamic equilibrium: (1) reading in a pressure-temperature table of precalculated
```

File: paper/paper.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+Natural Environment Research Council (U.K.) Large Grant MC-squared
+(Award No. NE/T012633/1). He would also like to thank M. Ghiorso for useful
+discussions and B. Watterson for the concept of Planet Zog.
+
+Timo Heister was partially supported by NSF Award DMS-1522191, DMS-1820958,
+OAC-1835452, by the Computational Infrastructure in Geodynamics initiative
+(CIG), through the NSF under Award EAR- 0949446 and EAR-1550901 and
+The University of California – Davis, and by Technical Data Analysis, Inc.
+through US Navy SBIR N16A-T003.
+
```
Comment (gassmoeller) [inline paper/paper.md:206]:
```suggestion
Rene Gassmoeller and Juliane Dannberg were supported by NSF Awards EAR-1925677 and
EAR-2054605, and by the Computational Infrastructure for Geodynamics (CIG) through the
NSF under Award EAR-0949446 and EAR-1550901 and the University of California – Davis.
```

File: paper/paper.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+so it is desirable to have well-tested and benchmarked software that
+provides convenient functions to calculate the properties of complex
+composite materials, and to fit new models.
+
+`BurnMan` is an open source, extensible mineral physics toolbox designed to
+calculate the physical properties of natural materials
+within the Earth and other planets. The toolbox has a class-based, modular
+design that allows users to calculate many low-level properties that
+are not accessible using existing codes, and to combine various tools in
+novel, creative ways.
```
Comment (gassmoeller) [inline paper/paper.md:78]: I think this paragraph should be part of the summary above (as a second paragraph). According to the JOSS website the summary should:
```
A summary describing the high-level functionality and purpose of the software for a diverse, non-specialist audience.
```
Which is exactly the contrent of this paragraph.

I think the rest of the text in the problem statement remains in good shape even if this paragraph is moved up.

Comment (bobmyhill) [reply paper/paper.md]: Thanks, I agree :)

File: paper/paper.md (hunk 1 above)
Comment (bobmyhill) [reply paper/paper.md]: Yes, definitely, thanks for the nudge.

**Review by jdannberg — APPROVED** [review]
Well, I can still approve anyway :smile:

**Review by sannecottaar — APPROVED** [review]

**Review by ian-r-rose — APPROVED** [review]

**Review by tjhei — APPROVED** [review]

**Pull request closed without merging**